src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AuthorizationApiServiceImpl.scala

```diff
@@ -64,11 +70,13 @@ final case class AuthorizationApiServiceImpl(jwtReader: JWTReader, sessionTokenG
     claims.getClaims.asScala.view.filterKeys(admittedSessionClaims.contains).toMap
   }
 
-  private def getOrganizationId(claims: JWTClaimsSet): Try[AnyRef] = for {
-    nullableOrgClaimsMap <- Try(claims.getJSONObjectClaim(organizationClaim)).as(MissingClaim(organizationClaim))
+  private def getOrganizationId(claims: JWTClaimsSet): Try[UUID] = for {
+    nullableOrgClaimsMap <- Try(claims.getJSONObjectClaim(organizationClaim))
```

Comment:
was it made on purpose? the semantic is pretty different, isn't it?

Reply:
`claims.getJSONObjectClaim` can return `Success(null)`, that's why I called the result `nullable`.
But feel free to suggest different naming!

Reply:
The thing I'm saying is that AFAIK `as` is an alias of `map(_ => )`, while here you're using `.leftMap(_ => )`. I trust that the unit tests will have catched any strange behaviour though.

Reply:
I feel your confusion 🥲 
That `as` function has been defined in the commons under `TryOps` as an alias for `leftMap`, your point is exactly why I replaced it.
IMO we should remove that function from the commons